packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -20,7 +20,8 @@
     let assetLogoUrl: string
     let chainName: string | undefined
 
-    $: $network, chainId, (chainName = getTooltipText())
+    $: network = $activeProfile.network
+    $: chainId, (chainName = getTooltipText())
```

Comment:
To have the reactivity as it was before we should add a condition here
```suggestion
    $: network = $activeProfile.network
    $: network, chainId, (chainName = getTooltipText())
```